docs/components/Date.md

```diff
@@ -0,0 +1,88 @@
+- **Min height - MinHeight** - Minimum height for all visible elements of the component to be shown. This is responsive to the height of the component, so no part of the component is ever cut off. This is the default value of the component's height property.
+
+### Output Sanitisation
+
+The output for the date component (Date.Value) is a DateValue comprising of the day, month and year inputs. There are exceptions to this, where the fields are not visible in the component.
+- If the year field is not visible (i.e. ShowYear is false), then the DateValue will use the default year set as an input property.
+- If the day field is not visible (i.e. ShowDay is false), then the DateValue will use the default day set as an input property.
+
+Note, the output sanitisation uses the Date formula. This means when a year is input that is less than 1900, the year output will be the year input + 1900.
+
+## Error Handling
+
+Below is a list of input cases that the date validation triggers an error for.
+
+- **All visible inputs are empty and the date input is required (i.e. Required input property is set to true)** - Triggers when the day, month and year fields are all visible and the user leaves all the inputs empty. For example, the input "Day: , Month: , Year: ".
+- **The number of fields with user input is at least 1 but less than the number of visible fields.** - Triggers when the number of fields with user input is greater than 1 but there is at least 1 other field that is visible and empty. For example, the inputs "Day: , Month: 3, Year: " and "Month: , Year: 1996".
+- **Day input is 0** - Triggers an error when the day field is visible and the user enters a day input of 0. For example, the input "Day: 0, Month: 3, Year: 1996".
+- **Day input is greater than 30 for a month input of 4, 6, 9 or 11** - Triggers an error when the day field is visible and a user enters a day input greater than 30 with a month input of 4, 6, 9 or 11. For example, the input "Day: 31, Month: 6, Year: 2005".
+- **Day input is greater than 31 for a month input of 1, 3, 5, 7, 8, 10 or 12** - Triggers an error when the day field is visible and a user enters a day input greater than 31 with a month input of 1, 3, 5, 7, 8, 10 or 12. For example, the input "Day: 32, Month: 1, Year: 2024".
+- **Day input is greater than 28 for a month input of 2 and a year input which is not a leap year** - Triggers when the day and year fields are visible and the user enters a day input greater than 28 for a month input of 2 and a year input which is not a leap year. For example, the input "Day: 29, Month: 2, Year: 1995".
+- **Day input is greater than 29 for a month input of 2 and a year input which is a leap year** - Triggers when the day and year fields are visible and the user enters a day input greater than 29 for a month input of 2 and a year input which is a leap year. For example, the input "Day: 30, Month:2, Year: 1996"
```

Comment:
It might be easier to combine all the "day input is greater than X" into a single one with something like "day is greater than the final day of the entered month and year, for example 31 April in any year or 29 February in a non-leap-year"